.circleci/config.yml

```diff
@@ -0,0 +1,74 @@
+# Python CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-python/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # specify the version you desire here
+      # use `-browsers` prefix for selenium tests, e.g. `3.6.1-browsers`
+      - image: circleci/python:2.7.15
+
+      # Specify service dependencies here if necessary
+      # CircleCI maintains a library of pre-built images
+      # documented at https://circleci.com/docs/2.0/circleci-images/
+      # - image: circleci/postgres:9.4
+
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "requirements.txt" }}
+          # fallback to using the latest cache if no exact match is found
+          - v1-dependencies-
+
+      - run:
+          name: install dependencies
+          command: |
+            sudo pip install -r requirements.txt
+            mkdir artifacts
```

Comment:
This looks a bit clunky.  According to the Lambda packaging docs (https://docs.aws.amazon.com/lambda/latest/dg/lambda-python-how-to-create-deployment-package.html) you should be able to do something like 'pip install -t . -r requirements.txt' to install all dependencies in the same directory as the sns_to_ses_mailer.py file and zip from the top-level of the project directory.